app/Models/PullRequest.php

```diff
@@ -30,4 +33,9 @@ public function user(): BelongsTo
     {
         return $this->belongsTo(User::class);
     }
+
+    public function canDelete(): Attribute
```

Comment:
Its good but in the future, lets make a trait to add every attributes and merge with append without replicate code.